JS/Lesson24/tugba/filter.js

```diff
@@ -344,6 +356,9 @@ const studentsExercise9 = [
 		grade: 'B-',
 	},
 ];
+const filteredStudents = studentsExercise9.filter(student => student.grade >= 'C');
```

Comment:
Unfortunately, comparing strings this way does not work in JS.

You can write a helper function that compares grades (check Lesson 15 assignment exercise 14) instead.